Tests/INCommonsTests/Swift/Extensions/Collection/Collection+OrderedIndex.swift

```diff
@@ -0,0 +1,60 @@
+import INCommons
+import XCTest
+
+final class Array: XCTestCase {
```

Comment:
Please name the file "Array+OrderedIndexTests.swift" and match the test case to the type with the "Tests" extension.
```suggestion
final class Array_OrderedIndexTests: XCTestCase {
```